docs/tutorial/maps.rst

```diff
@@ -181,748 +166,386 @@ Some basic statistical functions are built into Map objects:
 
-    import sunpy.map
-    import sunpy.data.sample
-    aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
-    aia_map.peek()
+.. code-block:: python
 
+    >>> my_map.center  # doctest: +REMOTE_DATA
+    <SkyCoord (Helioprojective: obstime=2011-06-07T06:33:02.770, rsun=696000.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-06-07T06:33:02.770, rsun=696000.0 km): (lon, lat, radius) in (deg, deg, m)
+        (-0.00406308, 0.04787238, 1.51846026e+11)>): (Tx, Ty) in arcsec
+        (3.22309951, 1.38578135)>
+    >>> my_map.bottom_left_coord  # doctest: +REMOTE_DATA
+    <SkyCoord (Helioprojective: obstime=2011-06-07T06:33:02.770, rsun=696000.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-06-07T06:33:02.770, rsun=696000.0 km): (lon, lat, radius) in (deg, deg, m)
+        (-0.00406308, 0.04787238, 1.51846026e+11)>): (Tx, Ty) in arcsec
+        (-1228.76466158, -1224.62447509)>
+    >>> my_map.top_right_coord  # doctest: +REMOTE_DATA
+    <SkyCoord (Helioprojective: obstime=2011-06-07T06:33:02.770, rsun=696000.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2011-06-07T06:33:02.770, rsun=696000.0 km): (lon, lat, radius) in (deg, deg, m)
+        (-0.00406308, 0.04787238, 1.51846026e+11)>): (Tx, Ty) in arcsec
+        (1235.21095899, 1227.39598836)>
 
-This creates a plot window with all axes defined, a plot title, and the image of the map data defined by the contents of the map.
-In non-interactive mode the plot window blocks execution and must be closed before doing anything else.
+But what if we wanted to know what pixel these physical coordinates correspond to?
+Thankfully, each Map has an associated World Coordinate System, or WCS, which is derived from the underlying metadata and expressed as an `astropy.wcs.WCS` object.
+The WCS is accessible as an attribute:
 
-plot()
-------
+.. code-block:: python
 
-For more advanced plotting the base sunpy objects also provide a `~sunpy.map.mapbase.GenericMap.plot` command.
-This command is similar to the pyplot `~matplotlib.pyplot.imshow` command in that it will create a figure and axes object for you if you haven't already.
+    >>> my_map.wcs  # doctest: +REMOTE_DATA
+    WCS Keywords
+    <BLANKLINE>
+    Number of WCS axes: 2
+    CTYPE : 'HPLN-TAN'  'HPLT-TAN'
+    CRVAL : 0.00089530541880571  0.00038493926472939
+    CRPIX : 512.5  512.5
+    PC1_1 PC1_2  : 0.99999706448085  0.0024230207763071
+    PC2_1 PC2_2  : -0.0024230207763071  0.99999706448085
+    CDELT : 0.00066744222222222  0.00066744222222222
+    NAXIS : 1024  1024
 
-When you create a plot with `~sunpy.map.GenericMap.peek` or
-`~sunpy.map.GenericMap.plot`, sunpy will use `astropy.visualization.wcsaxes` to
-represent coordinates on the image accurately, for more information see
-:ref:`wcsaxes-plotting`.
+WCS is a fairly complex topic, but all we need to know for now is that the WCS provides the transformation between the pixel coordinates of the image and physical or "world" coordinates.
```

Comment:
```suggestion
WCS is a fairly complex topic, but all we need to know for now is that the WCS provides the transformation back and forth between the pixel coordinates of the image and physical or "world" coordinates.
```
To be clear the transform is two way?

Reply:
I don't think "between" implies a preferred direction. "back and forth between" feels redundant. Furthermore, this bi-directionality is illustrated in the code below.

Reply:
is there somewhere we can point to explain WCS - even a paper or outside sunpy somewhere?

Reply:
There is an intersphinx link to `astropy.wcs.WCS` above which does include some explanatory docs on WCS. There are the WCS papers, but I would not say those are a gentle introduction to WCS. Maybe adding something like a call out box with some links would be helpful?